moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -105,6 +105,70 @@ void MotionPlanningFrame::sceneScaleChanged(int value)
           s->scale((double)value / 100.0);
           ps->getWorldNonConst()->addToObject(scaled_object_->id_, shapes::ShapeConstPtr(s),
                                               scaled_object_->shape_poses_[i]);
+
+          // get shape bound size
+          double half_bound = -std::numeric_limits<double>::max();
+          switch(s->type){
+            case shapes::ShapeType::SPHERE:{
+              shapes::Sphere* sphere = static_cast<shapes::Sphere*>(s);
+              half_bound = sphere->radius;
+              break;
+            }
+            case shapes::ShapeType::CYLINDER:{
+              shapes::Cylinder* cylinder = static_cast<shapes::Cylinder*>(s);
+              half_bound = (cylinder->radius > cylinder->length/2.0) ? cylinder->radius : cylinder->length/2.0;
```

Comment:
why not simply use `std::max(cylinder->radius, cylinder->length/2.0)`

Reply:
I agree with utilizing `std::max()` function.
But I will replace this function into `computeShapeBoundingSphere()`  as @rhaschke mentioned.